src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -2363,10 +2417,29 @@ public EntryResult tMedicationActivity2MedicationStatement(MedicationActivity cd
 
 		// indication -> reason
 		for (Indication indication : cdaMedicationActivity.getIndications()) {
-			Condition cond = tIndication2ConditionProblemListItem(indication, bundleInfo);
+			IEntryResult condResult = tIndication2ConditionEncounter(indication, localBundleInfo);
+
```

Comment:
Why did we switch categories here? And Isn't there a better category for medication actiivity indications.

Reply:
I think I was just copying and pasting code and messed up.